themes/classic/templates/catalog/_partials/category-footer.tpl

```diff
@@ -0,0 +1,33 @@
+{**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ *}
+<div id="js-product-list-footer">
+    {if $category.additional_description && $listing.pagination.items_shown_from == 1}
+        <div class="card">
+            <div class="card-block">
+                {$category.additional_description nofilter}
+            </div>
+        </div>
+    {/if}
+</div>
```

Comment:
```suggestion
{if $category.additional_description && $listing.pagination.items_shown_from == 1}
    <div id="js-product-list-footer">
        <div class="card">
            <div class="card-block">
                {$category.additional_description nofilter}
            </div>
        </div>
    </div>
{/if}
```

Reply:
@Progi1984 This is correct, the div has nothing to do with the description itself. Let people target it if they want. ;-)

Reply:
@Hlavtox This is not fixed.